docs/antrea-proxy.md

```diff
@@ -127,6 +128,61 @@ kube-proxy:
 * Do not create the `kube-proxy-windows` DaemonSet [when using Docker as the
   container runtime](windows.md#installation-via-wins-docker-based-runtimes)
 
+### Configuring load balancer mode for external traffic
+
+Starting with Antrea v1.13, the `defaultLoadBalancerMode` configuration
+parameter and the `service.antrea.io/load-balancer-mode` Service annotation
+can be used to specify how you want AntreaProxy to handle external traffic
+destined to LoadBalancerIPs and ExternalIPs of Services. Specifically, the mode
+determines how external traffic is processed when it's load balanced across
+Nodes. Currently, it has two options: nat (default) and dsr.
+
+* In NAT mode, external traffic is SNAT'd when it's load balanced across Nodes
+to ensure symmetric path. It's the default and the most applicable mode.
+
+* In DSR mode, external traffic is never SNAT'd and backend Pods running on
+Nodes that are not the ingress Node can reply to clients directly, bypassing
+the ingress Node. Therefore, traffic in DSR mode usually has lower latency,
+higher output bandwidth, and preserved client IP. Currently, it is applicable
+to Linux Nodes, encap mode, and IPv4 cluster. The feature gate
+`LoadBalancerModeDSR` must be enabled to use this mode for any Service.
+
+You can make the following changes to the `antrea-config` ConfigMap to specify
+load balancer mode for all Services:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-agent.conf: |
+    kubeAPIServerOverride: "<kube-apiserver URL>"
+    antreaProxy:
+      proxyAll: true
+      defaultLoadBalancerMode: <nat|dsr>
+```
+
+To configure a different load balancer mode for a particular Service, you can
+annotate the Service in the following way:
+
+```bash
+kubectl annotate service my-service service.antrea.io/load-balancer-mode=<nat|dsr>
+```
+
+**Note**: Configuring load balancer mode is only meaningful when `proxyAll` is
```

Comment:
```suggestion
**Note**: Configuring the load balancer mode is only meaningful when `proxyAll` is
```

Reply:
Like the header of the section, it's not emphasizing which load balancer mode but the behavior of configuring it, I remember it doesn't need "the" in this case but I'm not good at this. @antoninbas @jianjuns can you correct me?

Reply:
IMO, the version with "the" is better here, but it doesn't really impact understanding. It's more correct to have an article here, which could be "a" or "the".

Reply:
added "the"